src/pbt/v2/exceptions.py

```diff
@@ -0,0 +1,40 @@
+class DagNotAvailableException(ValueError):
+    def __init__(self, message):
+        self.message = message
+
+
+class DagFileDeletionFailedException(ValueError):
+    def __init__(self, message, e):
+        self.message = message
+        self.e = e
+
+
+class DagUploadFailedException(ValueError):
+    def __init__(self, message, e):
+        self.message = message
+        self.e = e
+
+
+class ProjectPathNotFoundException(ValueError):
```

Comment:
Shall we define an abstract PBTException instead of extending all of these from ValueError?